.copier-answers.yml

```
@@ -0,0 +1,13 @@
+_commit: v1.1.0
+_src_path: gh:superlinear-ai/substrate
+author_email: laurent@superlinear.eu
+author_name: Laurent Sorber <laurent@superlinear.eu>
+project_description: RAGLite is a Python toolkit for Retrieval-Augmented Generation
+    (RAG) with PostgreSQL or SQLite.
```

I would take this from `.cruft.json` (without the newline):
```suggestion
project_description: A Python toolkit for Retrieval-Augmented Generation (RAG) with SQLite or PostgreSQL.
```